db/schema.rb

```diff
@@ -242,11 +244,12 @@
     t.datetime "resolved_at"
     t.integer "resource_owner_id"
     t.string "access_code"
+    t.index ["application_id"], name: "fk_rails_4035c6e0ed"
```

Comment:
this seems to be a bit odd to push up @ashleyzhang, as this looks like a config?